src/View/mainpage/MainPage.py

```diff
@@ -396,6 +402,10 @@ def create_image_fusion_tab(self):
             self.image_fusion_view_sagittal, 0, 1)
         self.image_fusion_four_views_layout.addWidget(
             self.image_fusion_view_coronal, 1, 0)
+
+        self.image_fusion_four_views_layout.addWidget(
+            self.image_fusion_roi_transfer_option_view, 1, 1
```

Comment:
In regards the placement of the button. I feel that having the button in an area dedicated to viewing images, feels a bit disconnected with the purpose of this area. I feel that it might be better to have this button appear with the rest of the roi-related buttons in the structure tab? Or maybe somewhere in the toolbar as an icon.

Reply:
The position of the ROI Transfer button actually came from the product owner, our team had a discussion with him about it. The button's there to also limit the visibility of the button to only after image fusion is done. Therefore we believe that the button should be there at least for now.

Reply:
That is very fair.